### ISSUE TITLE
Add Feide implementation 

### ISSUE THREAD

**Issue by kotlarz** [webkom/lego#570]
This can be used for verifying student accounts and as an authentication method.

Low priority, should be added after release.

**Comment by odinuge** [comment]
WIP here: https://github.com/webkom/lego/pull/1146

(Partly) blocked by https://github.com/webkom/lego/issues/1368, since we should do some proper testing... :smile:
